orc8r/cloud/go/services/certifier/tokens.go

```diff
@@ -0,0 +1,102 @@
+package certifier
+
+import (
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"hash/crc32"
+	"strings"
+
+	"github.com/jxskiss/base62"
+)
+
+type TokenType string
+
+const (
+	Personal TokenType = "op"
+)
+
+// Length of checksum of token's byte array
+const checksumLength = 4
+
+// GenerateToken generates a random 32-byte token with a checksum
+func GenerateToken(tokenType TokenType) (string, error) {
+	// Generate 32 random bytes
+	bytes := make([]byte, 32)
+	_, err := rand.Read(bytes)
+	if err != nil {
+		return "", err
+	}
+
+	// Generate checksum bytes
+	checksum := crc32.ChecksumIEEE(bytes)
+	checksumBytes := i32tob(checksum)
+
+	// Combine to form final token
+	finalBytes := append(bytes, checksumBytes...)
+	token := base62.StdEncoding.EncodeToString(finalBytes)
```

Comment:
Why base62 instead of base64? Prefer base64 by default, but if there's a good reason for base62 then add comment to document why

Reply:
Perhaps not really a good reason, but I just mimicked  the Github tokens and the say a bit more about it: https://github.blog/changelog/2021-03-31-authentication-token-format-updates-are-generally-available/. I'll just change to base62 though, seems more standard.

Reply:
Actually a good reason for it is that I use it is bc I split by "_" and that is a base64 url safe character